ollama_proxy_server/main.py

```diff
@@ -77,6 +77,18 @@ def _send_response(self, response):
             self.end_headers()
             self.wfile.write(response.content)
 
+        def _send_response_stream(self, response):
+            self.send_response(response.status_code)
+            self.send_header('Content-type', response.headers['content-type'])
+            self.send_header('Stream', True)
+            self.end_headers()
+            for line in response.iter_lines():
+                if line:
+                    print(line)
```

Comment:
Please remove this or make it a configurable option. By default I don't want to supervise the users messages or the outputs.

Reply:
oh you mean the print? yes, i forgot to remove after my test. Will do it.